core/src/main/resources/hudson/model/Messages.properties

```diff
@@ -276,20 +276,33 @@ UpdateCenter.Status.ConnectionFailed=\
   <span class=error>Failed to connect to {0}. \
   Perhaps you need to <a href="../pluginManager/advanced">configure HTTP proxy?</a></span>
 UpdateCenter.init=Initialing update center
+UpdateCenter.PluginCategory.android=Android Development
 UpdateCenter.PluginCategory.builder=Build Tools
 UpdateCenter.PluginCategory.buildwrapper=Build Wrappers
 UpdateCenter.PluginCategory.cli=Command Line Interface
 UpdateCenter.PluginCategory.cluster=Cluster Management and Distributed Build
+UpdateCenter.PluginCategory.database=Database
+UpdateCenter.PluginCategory.deployment=Deployment
+UpdateCenter.PluginCategory.devops=DevOps
+UpdateCenter.PluginCategory.dotnet=.NET Development
 UpdateCenter.PluginCategory.external=External Site/Tool Integrations
+UpdateCenter.PluginCategory.groovy-related=Groovy-related
```

Comment:
It's not Groovy development though, these are plugins that use Groovy scripting in some way, e.g. Groovy Postbuild.